portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,30 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  public ArrayList<String> coelho;
```

Comment:
I was trying to figure out if this was a word I didn't know, but now I realize it's a name! And now the rest of the class makes more sense :P

Reply:
Oops - I'll be more clear with future naming, i.e. coelhoQuotes.